pandas/core/frame.py

```diff
@@ -663,7 +663,9 @@ def _repr_html_(self):
 
     @Substitution(header='Write out the column names. If a list of strings '
                          'is given, it is assumed to be aliases for the '
-                         'column names')
+                         'column names',
+                  col_space_type='int',
```

Comment:
same

Reply:
I added `versionadded` to `to_html`, but this one shouldn't need that as this is the same parameter and description as before.  I just had to add it to `Substitution` because the sections in `common_docstring` had to be parameterized to allow that section to be different on both functions.